src/tokens.ts

```diff
@@ -170,13 +164,43 @@ export namespace IDebugger {
     path: string;
   };
 
+  /**
+   * The type for a stack frame
+   */
+  export type IStackFrame = DebugProtocol.StackFrame;
```

Comment:
Maybe we should do `interface IStackFrame extends DebugProtocol.StackFrame{}` if we want the `I` at the beginning? I'm not sure what to do here, to be honest.

Reply:
Yes eslint formats it automatically. But we could disable the rule.

Reply:
Oh right, I forgot that constraint ... from yesterday :joy:

Hm, I think we should disable that rule if it's making programmatic decisions for us. What do you think?

Reply:
Sounds good :+1: 

Especially since core lab disables it too: https://github.com/jupyterlab/jupyterlab/blob/5e34867f355f63a5dcf5f77e6ff780f877b5748f/.eslintrc.js#L38